File: bronze/4153/c++/main.cpp
#include <iostream>

int main()
{
    int a, b, c, temp;

    while (true) {
        
        std::cin >> a >> b >> c;
        if (a == 0 && b == 0 && c == 0) {
            break;
        }

        if (c < a) {
            temp = c;
            c = a;
            a = temp;
        }
        if (c < b) {
            temp = c;
            c = b;
            b = temp;
        }

        std::cout << (((a*a) + (b*b) == (c*c)) ? "right\n" : "wrong\n");
    }

    return 0;
}

File: bronze/4153/python/main.py

if (__name__ == "__main__"):
    while (True):
        array = list(map(int, input().split()))

        if (array == [0, 0, 0]): break

        array.sort()

        if ((array[0]*array[0]) + (array[1]*array[1]) == (array[2]*array[2])):
            print("right")
        else:
            print("wrong")
        